easybuild/easyconfigs/b/Blender/Blender-2.66a-foss-2016.04-trimmed.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'Blender'
+version = '2.66a'
+versionsuffix = '-trimmed'
+
+homepage = 'https://www.blender.org/'
+description = """Blender is the free and open source 3D creation suite. It supports
+ the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,
+ compositing and motion tracking, even video editing and game creation."""
+
+toolchain = {'name': 'foss', 'version': '2016.04'}
+
+sources = ['blender-2.66a.tar.gz']
+source_urls = ['http://download.blender.org/source/']
+
+# These are needed until extra dependencies are added to flesh it out 
+configopts = '-DWITH_INSTALL_PORTABLE=OFF '
+configopts += '-DWITH_BUILDINFO=OFF '
+configopts += '-DWITH_GAMEENGINE=OFF '
+configopts += '-DWITH_CYCLES=OFF '
+configopts += '-DWITH_SYSTEM_GLEW=OFF '
+
+dependencies = [
+    ('Python', '3.3.2'),
```

Comment:
when Python is listed as a dependency, we generally label the easyconfig with a versionsuffix like `-Python-%(pyver)s` (where `%(pyver)s` is a template value that EasyBuild will fill in automagically)

this makes it clear that only modules using the same Python version will be compatible with this modules (just because Python is such a common dep)